src/webui/components/Login/index.js

```diff
@@ -32,20 +51,30 @@ export default class LoginModal extends Component {
    * set login modal's username and password to current state
    * Required by: <LoginModal />
    */
-  setCredentials(name, e) {
+  setCredentials(key, value) {
     this.setState({
-      [name]: e
+      form: Object.assign({}, this.state.form, {[key]: value})
     });
   }
 
-  async submitCredentials(event) {
-    // prevents default submit behaviour
+  /**
+   * Clears the username and password field.
+   */
+  handleReset() {
+    this.formRef.current.resetFields();
```

Comment:
Here, imho I'd rather reset only the password, username after auth fails should remains.

Reply:
@priscilawebdev any thought here?

Reply:
@juanpicado @ayusharma I just checked how it works in Gmail :) It's as Juan said ... I agree with him.